sycl/test-e2e/Regression/empty_accessor_use.cpp

```diff
@@ -0,0 +1,25 @@
+// RUN: %{build} -o %t.out
+// RUN: %{run} %t.out
+
+// Tests that 3D accessors with 0 elements are allowed to be captured in a
+// kernel.
+
+#include <sycl/sycl.hpp>
+
+using namespace sycl;
+
+int main() {
+  queue Q;
+
+  buffer<int, 3> Buf{range<3>{1, 1, 1}};
+  assert(Buf.size() == 1);
+  Q.submit([&](handler &CGH) {
+    accessor Acc{Buf, CGH, range<3>{0, 0, 0}, read_write};
+    // assert(Acc.empty());
```

Comment:
Uncomment or add a comment when it will be enabled?

Reply:
Ah, you're right! I'm not sure why I commented it out.
```suggestion
    assert(Acc.empty());
```